code/modules/food/food/snacks.dm

```diff
@@ -83,9 +84,23 @@
 				to_chat(M, "<span class='notice'>You take a bite of [src].</span>")
 			if (fullness > 350 && fullness <= 550)
 				to_chat(M, "<span class='notice'>You unwillingly chew a bit of [src].</span>")
-			if (fullness > (550 * (1 + M.overeatduration / 2000)))	// The more you eat - the more you can eat
-				to_chat(M, "<span class='danger'>You cannot force any more of [src] to go down your throat.</span>")
+			if (fullness > 550 && fullness <= 650)
+				to_chat(M, "<span class='notice'>You swallow some more of the [src], causing your belly to swell out a little.</span>")
+			if (fullness > 650 && fullness <= 1000)
+				to_chat(M, "<span class='notice'>You stuff yourself with the [src]. Your stomach feels very heavy.</span>")
+			if (fullness > 1000 && fullness <= 3000)
+				to_chat(M, "<span class='notice'>You gluttonously swallow down the hunk of [src]. You're so gorged, it's hard to stand.</span>")
+			if (fullness > 3000 && fullness <= 5500)
+				to_chat(M, "<span class='danger'>You force the piece of [src] down your throat. You can feel your stomach getting firm as it reaches its limits.</span>")
```

Comment:
👀 i feel like we should keep this stuff to be vore-specific

Reply:
Oh uh, woops. I forgot to edit that out. I started coding this downstream. Figured what i'm trying to code isn't too specific for the codebase that I might as well code it here. Lemme fix this.